mungegithub/mungers/issue-categorizer.go

```diff
@@ -78,38 +77,6 @@ func (lm *LabelMunger) Munge(obj *github.MungeObject) {
 	if len(tLabels) == 0 && len(cLabels) == 0 {
 		obj.AddLabels(getRoutingLabels(lm.triagerUrl, obj.Issue.Title, obj.Issue.Body))
 	}
-	// else {
```

Comment:
All of this was totally dead code. I would have needed to pass the bot's name into this config as well if we wanted to support the methods that are called in this comment block... but someone can do that when they bring this out of cold storage, maybe.